Exebite.DataAccess/Repositories/CommentRepository/ICommentCommandRepository.cs

```diff
@@ -1,8 +1,12 @@
-﻿using Exebite.DataAccess.Repositories.CommentRepository.Model;
+﻿using Either;
+using Exebite.Common;
+using Exebite.DataAccess.Repositories.CommentRepository.Model;
+using Exebite.DtoModels.Comment;
 
 namespace Exebite.DataAccess.Repositories.CommentRepository
 {
     public interface ICommentCommandRepository : IDatabaseCommandRepository<long, CommentInsertModel, CommentUpdateModel>
     {
+        Either<Error, bool> Delete(int commentId, DeleteCommentDto comment);
```

Comment:
Consequently, if we were to modify the delete comment endpoint route, this method would look like following:

```suggestion
        Either<Error, bool> Delete(int commentId, long mealId);
```